webpack.config.js

```diff
@@ -88,14 +91,24 @@ module.exports = (env, argv) => { // eslint-disable-line no-undef
                     test: /\.css/,
                     exclude: /ckeditor5-[^/\\]+[/\\]theme[/\\].+\.css$/,
                     use: [
-                        MiniCssExtractPlugin.loader,
+                        {
+                            loader: MiniCssExtractPlugin.loader,
+                            options: {
+                                publicPath: '../../', // @see https://github.com/sulu/sulu/pull/6225
```

Comment:
we configure the `MiniCssExtractPlugin` to output the bundled css into `build/admin/main.HASH.css`:
https://github.com/sulu/sulu/blob/1b6837b2ca7f64f8baf833508f1a7bb0c7ff0630/webpack.config.js#L50-L52

when we reference another resource inside of a `.scss` file, the path is replaced with the public path returned by the respective loader. for example, the `./tablet.png` is loaded by the `file-loader` and replaced with `build/admin/images/tablet.HASH.png`:
https://github.com/sulu/sulu/blob/602d71aa0c23b864b9d0de06be78d26f5e9a483d/src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/preview.scss#L30

this will look something like this i the bundled `build/admin/main.HASH.css`:
```
background-image: url(build/admin/images/tablet.HASH.png);
```

as the browser resolves `url()` inside of a stylesheet relative to the location of the stylesheet, the browser tries to resolve the url relative to the `build/admin/main.HASH.css` instead of the correct url:
```
WRONG URL:      build/admin/build/admin/images/tablet.HASH.png
CORRECT URL:    build/admin/images/tablet.HASH.png
```

in short, the problem is that the `file-loader` returns the correct public url for the `tablet.png`, but the `MiniCssExtractPlugin.loader` does not adjust this url to be relative to the location of the bundled css.

to fix this, we need to tell the `MiniCssExtractPlugin.loader` where to find the public path of the project relative to the bundled css by setting the `publicPath` option. after doing this, the `MiniCssExtractPlugin.loader` will adjust the url returned by the `file-loader` and the bundled css will look something like this:
```
background-image: url(../../build/admin/images/tablet.HASH.png);
```

the following issue might be helpful for understanding whats happening too: https://github.com/webpack-contrib/mini-css-extract-plugin/issues/44#issuecomment-391009221